langchain4j/src/main/java/dev/langchain4j/store/embedding/chroma/QueryRequest.java

```diff
@@ -0,0 +1,52 @@
+package dev.langchain4j.store.embedding.chroma;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+
+class QueryRequest {
+
+    private final List<List<Float>> query_embeddings;
+    private final int n_results;
+    private final List<String> include;
+
+    public QueryRequest(Builder builder) {
+        this.query_embeddings = builder.queryEmbedding;
+        this.n_results = builder.nResults;
+        this.include = builder.include;
+    }
+
+    @Override
+    public String toString() {
+        return "QueryRequest{" +
+                " queryEmbedding=" + query_embeddings +
+                ", nResults=" + n_results +
+                ", include=" + include +
+                '}';
+    }
+
+    public static Builder builder() {
+        return new Builder();
+    }
+
+    public static class Builder {
+        private List<List<Float>> queryEmbedding;
+        private int nResults = 10;
+        private List<String> include = asList("metadatas", "documents", "distances", "embeddings");
+
+        public Builder queryEmbedding(List<List<Float>> queryEmbedding) {
```

Comment:
please change this method to accept List<Float> instead

Reply:
changed